src/payment/offsite-payment-mappers/payload-mapper.js

```diff
@@ -79,6 +79,7 @@ export default class PayloadMapper {
                 payment_method_id: paymentMethod.id,
                 reference_id: order.orderId ? toString(order.orderId) : null,
                 return_url: paymentMethod.returnUrl || (order.payment ? order.payment.returnUrl : null),
+                should_save_instrument: payment.shouldSaveInstrument ? payment.shouldSaveInstrument : null,
```

Comment:
The name of this field in the API is `vault_payment_instrument`, could we keep it consistent?

https://github.com/bigcommerce/bigpay-client-js/blob/ab24e7ad67e05701a2a7220e8e8e84dea011b1d8/src/payment/v1/payment-mappers/payment-mapper.js#L46